### DISCUSSION TITLE
The school you selected does not appear to have a campus location in your country

### DISCUSSION THREAD

**Discussion by asherbo123** [community/community#31927] [GitHub Education]
I can't use student benefits because of this error.
Your application cannot be reviewed until you fix the following:
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

**Comment by MathiasReker** [comment]
you need to contact your school about this.

**Comment by Osczrrs** [comment]
yeah im having the same issue any help?

**Reply by wubaosheng** [reply]
same issues
